ui/webui/test/helpers/storage.py

```diff
@@ -15,16 +15,30 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program; If not, see <http://www.gnu.org/licenses/>.
 
+import os
+import sys
+
+HELPERS_DIR = os.path.dirname(__file__)
+sys.path.append(HELPERS_DIR)
+
+from installer import InstallerSteps  # pylint: disable=import-error
+
+
 STORAGE_INTERFACE = "org.fedoraproject.Anaconda.Modules.Storage"
 STORAGE_OBJECT_PATH = "/org/fedoraproject/Anaconda/Modules/Storage"
 
+
 class Storage():
     def __init__(self, browser, machine):
         self.browser = browser
         self.machine = machine
+        self._step = InstallerSteps.storage
 
     def select_disk(self, disk, selected=True):
-        self.browser.set_checked("#" + disk + " input", selected)
+        self.browser.set_checked(f"#{disk} input", selected)
+
+    def disk_selected(self, disk):
```

Comment:
We prefer function/method names that start with a verb to avoid a confusion with attributes and properties.